homeassistant/components/incomfort/climate.py

```diff
@@ -94,18 +85,18 @@ def supported_features(self) -> int:
     @property
     def min_temp(self) -> float:
         """Return max valid temperature that can be set."""
-        return INTOUCH_MIN_TEMP
+        return 5.0
 
     @property
     def max_temp(self) -> float:
         """Return max valid temperature that can be set."""
-        return INTOUCH_MAX_TEMP
+        return 30.0
 
-    async def async_set_temperature(self, **kwargs) -> None:
+    async def async_set_temperature(self, **kwargs) -> Awaitable[None]:
         """Set a new target temperature for this zone."""
         temperature = kwargs.get(ATTR_TEMPERATURE)
         await self._room.set_override(temperature)
 
-    async def async_set_hvac_mode(self, hvac_mode: str) -> None:
+    async def async_set_hvac_mode(self, hvac_mode: str) -> Awaitable[None]:
```

Comment:
simple `None`, there is nothing available on return...

Reply:
You guys are machines!